applications/osb-portal/src/components/workspace/drawer/WorkspaceInteractions.tsx

```diff
@@ -192,7 +193,7 @@ export default (props: WorkspaceProps | any) => {
             </Typography>
 
             <Box p={2}>
-              <WorkspaceActionsMenu workspace={workspace} user={props.user} updateWorkspace={props.updateWorkspace} deleteWorkspace={props.deleteWorkspace} refreshWorkspaces={props.refreshWorkspace} />
+              <WorkspaceActionsMenu workspace={workspace} user={props.user} updateWorkspace={props.updateWorkspace} deleteWorkspace={props.deleteWorkspace} refreshWorkspaces={props.refreshWorkspace} cloneWorkspace={props.cloneWorkspace} />
```

Comment:
Just for completeness: here we could have done the redux props mapping but wasn't done from the beginning. No choice is correct of wrong by itself, but I think it's good to keep consistency by having either having the component connected to redux or not, and if it is, using the connector to bind the actions. as props

Reply:
Since this component is not connected to the redux store yet, I've left this as is for the moment.